ocrd/ocrd/cli/network.py

```diff
@@ -23,9 +23,6 @@ def network_cli():
     Managing network components
     """
     initLogging()
```

Comment:
They are required for the CLI because without it, no logging is set up.

The idea is to only use `initLogging` if we're in "application mode" not in "library mode" where there might be logging set up in a different way. E.g. in ODEM, ocrd_kitodo or OPERANDI, there likely is a separate logging setup, with which we don't want to interfere.